### PR TITLE
acceptance tests for newly registered clients

### PR DESCRIPTION
**Pull request by individual-it**
tests to check if a new client can be registered via webUI and if the new client-id and client-secret can be used to obtain tokens

depends on https://github.com/owncloud/core/pull/31412 and https://github.com/owncloud/core/pull/31420 being merged and new QA builds made

### CODE REVIEW

File: tests/acceptance/features/webUIOauth2/newClientRegistration.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	I want to be able to register new clients
+	So that private set of client_ids and client_secrets can be used
+
+	Scenario: register a new client on the webUI
+		Given user admin has logged in using the webUI
+		And the administrator has browsed to the oauth admin settings page
+		When the administrator adds a new oauth client with the name "new client" and the uri "http://localhost:*" using the webUI
+		Then a new client with the name "new client" and the uri "http://localhost:*" should be listed on the webUI
+
+	Scenario: oath authorization with a new client
```
Comment (phil-davis) [inline tests/acceptance/features/webUIOauth2/newClientRegistration.feature]: s/oath/oauth

File: tests/acceptance/features/lib/Oauth2AdminSettingsPage.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+	/**
+	 * 
+	 * @param string $name
+	 * 
+	 * @throws ElementNotFoundException
+	 * 
+	 * @return string[] arrray with keys name,redirection_uri,client_id,client_secret,id
+	 */
+	public function getClientInformationsByName($name) {
```
Comment (phil-davis) [inline tests/acceptance/features/lib/Oauth2AdminSettingsPage.php]: "information" is a singular thing in English/
Change function name to ``getClientInformationByName``

**Comment by SamuAlfageme** [comment]
@individual-it @phil-davis now that you guys are looking at these auto-tests; could you add some scenarios I missed like the one described in https://github.com/owncloud/oauth2/issues/135 to the server testplan from https://github.com/owncloud/QA/pull/498? (i.e. validations, missing fields, etc.)

Awesome job, btw

**Comment by individual-it** [comment]
Test passing. please review again

**Pull request merged**